src/server/JasmineGraphServer.cpp

```diff
@@ -2360,6 +2365,1064 @@ int JasmineGraphServer::removePartitionThroughService(string host, int port, str
     return 0;
 }
 
+int JasmineGraphServer::initiateEntityResolution(vector<pair<string, string>> hostHasPartition, string graphID,
+                                                 std::string masterIP, string designatedWorkerHost,
+                                                 int designatedWorkerPort) {
+    bool isCoordinator = false;
+
+    //Identify coordinating worker
+    string coordinatorHost;
+    int coordinatorPort;
+    if (isCoordinator) {
+        coordinatorHost = designatedWorkerHost;
+        coordinatorPort = designatedWorkerPort;
+    } else {
+        coordinatorHost = hostHasPartition.at(0).first;
+        coordinatorPort = hostPortMap[coordinatorHost].first;
+    }
+
+    //Signal other organizations to start ER process
+    vector<string> orgList;
+    if (isCoordinator) {
+        int orgCount;
```

Comment:
Here `orgCount` variable is not initialized to a value. And we are using that variable in the next line as,

`std::thread *orgThreads = new std::thread[orgCount];`

We should initialize `orgCount` to a value before using it.

Reply:
Value initialized

Reply:
Ok